dm/tests/adjust_gtid/run.sh

```diff
@@ -84,10 +84,13 @@ function run() {
 	# avoid cannot unmarshal !!str `binlog-...` into uint32 error
 	sed -i "s/binlog-pos-placeholder-1/4/g" $WORK_DIR/dm-task.yaml
 	sed -i "s/binlog-pos-placeholder-2/4/g" $WORK_DIR/dm-task.yaml
-	dmctl_start_task "$WORK_DIR/dm-task.yaml" "--remove-meta"
+	# start DM task. don't check error because it will meet injected error soon
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"start-task $WORK_DIR/dm-task.yaml --remove-meta"
 
 	# use sync_diff_inspector to check full dump loader
-	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
```

Comment:
seems we don't need  `diff_config.toml`, can we delete that file?

Reply:
we'll use it after sync_diff fix the behaviour soon